content/books/_index.es.md

```
@@ -0,0 +1,33 @@
+- **2021** – {{< book-count 2021 >}}, {{< page-count 2021 >}}
+- **2020** – {{< book-count 2020 >}}, {{< page-count 2020 >}}
+- **2019** – {{< book-count 2019 >}}, {{< page-count 2019 >}}
+- **2018** – {{< book-count 2018 >}}, {{< page-count 2018 >}}
+- **2017** – {{< book-count 2017 >}}, {{< page-count 2017 >}}
+- **2016** – {{< book-count 2016 >}}, {{< page-count 2016 >}}
+- **2015** – {{< book-count 2015 >}}, {{< page-count 2015 >}}
+- **2014** – {{< book-count 2014 >}}, {{< page-count 2014 >}}
+
+</details>
+
+Más abajo encontrarás mis últimas lecturas con sus reseñas y comentarios. También puedes [verlas en formato cuadrícula](/media/books/grid).
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)